File: app/src/main/java/ru/ra66it/updaterforspotify/domain/models/FullSpotifyModel.java
package ru.ra66it.updaterforspotify.domain.models;

/**
 * Created by 2Rabbit on 14.01.2018.
 */

public class FullSpotifyModel {

    private String latestLink;
    private String latestVersionName;
    private String latestVersionNumber;

    public FullSpotifyModel(Spotify spotify) {
        latestLink = spotify.getData().getFile().getPath();
        latestVersionName = spotify.getData().getName() + " " + spotify.getData().getFile().getVername();
        latestVersionNumber = spotify.getData().getFile().getVername();
    }

    public String getLatestLink() {
        return latestLink;
    }

    public String getLatestVersionName() {
        return latestVersionName;
    }

    public String getLatestVersionNumber() {
        return latestVersionNumber;
    }
}

File: app/src/main/java/ru/ra66it/updaterforspotify/domain/models/FullSpotifyModel.kt
package ru.ra66it.updaterforspotify.domain.models

/**
 * Created by 2Rabbit on 14.01.2018.
 */

class FullSpotifyModel(spotify: Spotify) {

    val latestLink: String = spotify.data?.file?.path!!
    val latestVersionName: String = spotify.data?.name + " " + spotify.data?.file?.vername
    val latestVersionNumber: String = spotify.data?.file?.vername!!

}
